release/BUILD

```diff
@@ -5,10 +5,10 @@ compile_pip_requirements(
     name = "requirements_buildkite",
     requirements_in = "requirements_buildkite.in",
     requirements_txt = "requirements_buildkite.txt",
-    visibility = ["//visibility:private"],
     tags = [
-            "team:core",
-        ],
+        "team:ci",
```

Comment:
Is the team name just for decoration or it means something (e.g. oncall, etc.)?

Reply:
I think this is used to filter in https://flakey-tests.ray.io/?owner=ci